consensus/istanbul/backend/backend_test.go

```diff
@@ -0,0 +1,352 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package backend
+
+import (
+	"github.com/hashicorp/golang-lru"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	istanbulCore "github.com/klaytn/klaytn/consensus/istanbul/core"
+	"github.com/klaytn/klaytn/consensus/istanbul/validator"
+	"github.com/klaytn/klaytn/crypto"
+	"github.com/klaytn/klaytn/event"
+	"github.com/klaytn/klaytn/governance"
+	"github.com/klaytn/klaytn/networks/p2p"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/reward"
+	"github.com/klaytn/klaytn/storage/database"
+	"math/big"
+	"testing"
+)
+
+var (
```

Comment:
```suggestion
const (
```

Reply:
@jeonghonah Sorry. Slice and arrays can't be a constant in nature in Golang. I'll revert it back to `var`